domain-ee/ee-ep-merge-app/src/requirements.txt

```diff
@@ -0,0 +1,10 @@
+fastapi==0.103.*
+hoppy @ git+https://github.com/department-of-veterans-affairs/abd-vro.git@dfitchett/hoppy_async_client#subdirectory=shared/lib-hoppy
```

Comment:
This needs to be changed once #2044 is merged into develop, and the commit is tagged with the hoppy version.